docs/user/adaptive-learning/adaptive-learning-student.rst

```diff
@@ -21,10 +24,11 @@ Expanding the prerequisites section shows the student all competencies the instr
 |students-learning-goals-statistics|
 
 When clicking on a competency, a page opens and displays detailed statistics about the competency together with all linked lecture units and exercises.
-The tripartite rings show the student's advancement:
-The **blue ring describes the progress**, the percentage of completed lecture units and exercises.
-The **green ring visualizes the confidence**, the average score in all linked exercises in relation to the threshold required for mastering the competency (set by the instructor).
-The **red ring is a weighted metric of the student's progress and confidence**, which shows the overall advancement toward competency mastery.
+The bipartite rings show the student's advancement:
+The **green ring describes the progress**, the percentage of completed lecture units and exercises.
+The **red ring shows the mastery**, which shows the overall advancement toward competency completion.
```

Comment:
(optional) You can change the verb in the first sentence to avoid duplication: The red ring indicates/describes mastery ...